src/components/attendee/VideoComms/scss/VideoCommsParticipant.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "scss/constants";
```

Comment:
Many values in this file can be commented for not using SCSS constants but raw `px`

Reply:
Agreed. I'll address this properly in the redesign.